### PR TITLE
Update version

### PR DESCRIPTION
**Pull request by chenrui333**
- Simplify some travis build process
- Ensure download the latest terraform version
- Add some flexibility to the `required_version`

### CODE REVIEW

**Comment by chenrui333** [comment]
@max-rocket-internet @brandoconnor Can I get a review on this? Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -17,12 +23,8 @@ install:
...
 before_script:
-- echo 'before_script'
-- export AWS_REGION='us-east-1'
-- export TF_VAR_region=${AWS_REGION}
-- echo "using AWS_REGION=${AWS_REGION}"
-- export TF_WARN_OUTPUT_ERRORS=1
-- curl --silent --output terraform.zip https://releases.hashicorp.com/terraform/0.11.8/terraform_0.11.8_linux_amd64.zip
+- export LATEST_TERRAFORM_VERSION=$(curl -s https://api.github.com/repos/hashicorp/terraform/releases/latest | grep tag_name | cut -d '"' -f 4 | cut -c2-)
+- curl --silent --output terraform.zip https://releases.hashicorp.com/terraform/${LATEST_TERRAFORM_VERSION}/terraform_${LATEST_TERRAFORM_VERSION}_linux_amd64.zip
 - sha256sum terraform.zip  | grep "84ccfb8e13b5fce63051294f787885b76a1fedef6bdbecf51c5e586c9e20c9b7"
```
Comment (brandoconnor) [inline .travis.yml]: I gather this line will start failing as we dynamically pull the latest terraform. If there's a way to also dynamically pull the checksum, I'm all for doing that. If that's too burdensome, I'd be okay to scrap the checksum altogether.

**Comment by brandoconnor** [comment]
One comment needing a change but otherwise it LGTM @chenrui333 . At my current gig I've been doing a lot of this test work within containers and others have asked for similar in this repo. Expect to see that added over the next month (I'll be on vacation for a bit).

File: .travis.yml (hunk 1 above)
Comment (chenrui333) [reply .travis.yml]: that is good catch, I can remove this checksum, unless you have other thought.

**Comment by sandipan6d** [comment]
Looks like build failed because it failed to get  LATEST_TERRAFORM_VERSION. You may like to use like 
```
LATEST_TERRAFORM_VERSION=$(curl -s https://api.github.com/repos/hashicorp/terraform/releases/latest | grep tag_name | cut -d":" -f2 | sed 's/"//g' | sed 's/,//g' | sed 's/ //g')
```

**Comment by chenrui333** [comment]
@sandipan6d It looks like my script has some `cut` issue:
```
$ curl -s https://api.github.com/repos/hashicorp/terraform/releases/latest | grep tag_name | cut -d '"' -f 4 | cut -c2-
ttps://api.github.com/repos/hashicorp/terraform/releases/13610319

$ curl -s https://api.github.com/repos/hashicorp/terraform/releases/latest | grep tag_name | cut -d '"' -f 4 | cut -c1-
https://api.github.com/repos/hashicorp/terraform/releases/13610319
```

I forgot why I use `cut -c1-`

**Comment by chenrui333** [comment]
I will submit a fix PR to see how it goes.

**Pull request merged**